backend/database-collection-schemas/createAnnouncementCollection.py

```diff
@@ -124,7 +138,10 @@
                 "updated_at": d[0],
                 "title": d[1],
                 "content": d[2],
+                "creator": d[3],
```

Comment:
also removed dummy data creation from script, since they are not needed for production

Reply:
That's okay.